packages/components/addon/components/hds/dropdown/index.hbs

```diff
@@ -0,0 +1,38 @@
+<Hds::Disclosure class="hds-dropdown">
+  <:toggle as |t|>
+    {{#if (eq @toggle "toggleButton")}}
+      <Hds::Dropdown::ToggleButton
+        @color={{@color}}
+        @isOpen={{t.isActive}}
+        @text={{@text}}
+        {{on "click" t.onClickToggle}}
+      />
+    {{else}}
+      <Hds::Dropdown::ToggleIcon
+        @hasChevron={{@hasChevron}}
+        @icon={{@icon}}
+        @imageSrc={{@imageSrc}}
+        @isOpen={{t.isActive}}
+        @text={{@text}}
+        {{on "click" t.onClickToggle}}
+      />
+    {{/if}}
+  </:toggle>
+  <:content>
+    <ul
+      class="hds-dropdown-list
+        {{if (eq @listPosition 'left') 'hds-dropdown-list--position-left' 'hds-dropdown-list--position-right'}}"
+    >
+      {{yield
+        (hash
+          CopyItem=(component "hds/dropdown/list-item" item="copy-item")
+          Description=(component "hds/dropdown/list-item" item="description")
+          Generic=(component "hds/dropdown/list-item" item="generic")
+          Interactive=(component "hds/dropdown/list-item")
```

Comment:
You could do this and remove the default from the backing class, more clear (and semantically correct, now that we are yielding the components there's no more "default").
```suggestion
          Interactive=(component "hds/dropdown/list-item" item="interactive")
```